### ISSUE TITLE
[Bug]: v25.0.1 - Depending on screen size, URL sharing details options may disappear from the screen and not be selectable

### ISSUE THREAD

**Issue by tatsuyasenbom** [nextcloud/server#35636]
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [X] This issue is **not** already reported on Github _(I've searched it)_.
- [X] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [X] Nextcloud Server **is** running on 64bit capable CPU, PHP and OS.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

1. In the following environment :
    * ![version](https://user-images.githubusercontent.com/87968771/206059343-8baf23ad-c329-4961-9475-f2661b2d5e23.png)
    * ![spec1](https://user-images.githubusercontent.com/87968771/206059529-5d756398-267c-4755-a236-73956597c76a.png)
    * ![spec2](https://user-images.githubusercontent.com/87968771/206059600-9238dab0-18be-4901-b0cf-8e2a74d24649.png)
2. I have identified the following phenomena.
    * ![bug](https://user-images.githubusercontent.com/87968771/206059024-344da342-a553-4701-bc71-c29b855bdb97.png)
3. I am unable to select the popup option that appears when I press "・・・" on "URLを共有" because it is missing from the screen.
4. If the screen resolution is large, the option may be selected without disappearing.

### Steps to reproduce

* I am unable to select the popup option that appears when I press "・・・" on "URLを共有" because it is missing from the screen.
    * ![bug](https://user-images.githubusercontent.com/87968771/206059024-344da342-a553-4701-bc71-c29b855bdb97.png)

### Expected behavior

* Is it possible to make the pop-up option that appears when I press "・・・" on "URLを共有" selectable without it disappearing from the screen?

### Installation method

Community Manual installation with Archive

### Operating system

RHEL/CentOS

### PHP engine version

PHP 8.0

### Web server

Apache (supported)

### Database engine version

MySQL

### Is this bug present after an update or on a fresh install?

Updated to a major version (ex. 22.2.3 to 23.0.1)

### Are you using the Nextcloud Server Encryption module?

Encryption is Disabled

### What user-backends are you using?

- [X] Default user-backend _(database)_
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### Configuration report

```shell
[root]# sudo -u apache php occ config:list system
{
    "system": {
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "overwriteprotocol": "https",
        "default_phone_region": "JP",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "nextcloud.test.com"
        ],
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "dbtype": "mysql",
        "version": "25.0.1.1",
        "overwrite.cli.url": "https:\/\/nextcloud.test.com",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbport": "",
        "dbtableprefix": "oc_",
        "mysql.utf8mb4": true,
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "memcache.distributed": "\\OC\\Memcache\\Redis",
        "memcache.locking": "\\OC\\Memcache\\Redis",
        "memcache.local": "\\OC\\Memcache\\APCu",
        "redis": {
            "host": "***REMOVED SENSITIVE VALUE***",
            "port": 6379
        },
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpmode": "smtp",
        "mail_sendmailmode": "smtp",
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpport": "25",
        "maintenance": false,
        "theme": "",
        "loglevel": 2,
        "updater.release.channel": "stable",
        "logtimezone": "UTC"
    }
}
```

### List of activated Apps

```shell
[root]# sudo -u apache php occ app:list
Enabled:
  - activity: 2.17.0
  - circles: 25.0.0
  - cloud_federation_api: 1.8.0
  - comments: 1.15.0
  - contactsinteraction: 1.6.0
  - dashboard: 7.5.0
  - dav: 1.24.0
  - federatedfilesharing: 1.15.0
  - federation: 1.15.0
  - files: 1.20.1
  - files_automatedtagging: 1.15.0
  - files_pdfviewer: 2.6.0
  - files_retention: 1.14.0
  - files_rightclick: 1.4.0
  - files_sharing: 1.17.0
  - files_trashbin: 1.15.0
  - files_versions: 1.18.0
  - firstrunwizard: 2.14.0
  - groupfolders: 13.1.0
  - logreader: 2.10.0
  - lookup_server_connector: 1.13.0
  - nextcloud_announcements: 1.14.0
  - notifications: 2.13.1
  - oauth2: 1.13.0
  - password_policy: 1.15.0
  - photos: 2.0.0
  - privacy: 1.9.0
  - provisioning_api: 1.15.0
  - recommendations: 1.4.0
  - related_resources: 1.0.3
  - serverinfo: 1.15.0
  - settings: 1.7.0
  - sharebymail: 1.15.0
  - support: 1.8.0
  - survey_client: 1.13.0
  - systemtags: 1.15.0
  - text: 3.6.0
  - theming: 2.0.1
  - twofactor_backupcodes: 1.14.0
  - updatenotification: 1.15.0
  - user_retention: 1.8.1
  - user_status: 1.5.0
  - viewer: 1.9.0
  - weather_status: 1.5.0
  - workflowengine: 2.7.0
Disabled:
  - admin_audit
  - bruteforcesettings
  - encryption
  - files_external
  - suspicious_login
  - twofactor_totp
  - user_ldap
```

### Nextcloud Signing status

```shell
<html><head><meta name="color-scheme" content="light dark"></head><body><pre style="word-wrap: break-word; white-space: pre-wrap;">No errors have been found.</pre></body></html>
```

**Comment by szaimen** [comment]
This will be fixed with 25.0.2

**Comment by tatsuyasenbom** [comment]
> This will be fixed with 25.0.2

Thank you for your response.

I understand that this may be improved in the next minor version of the software.

Thank you!
